app/components/Question/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+/* component styles */
+import { styles } from './styles.scss';
+
+const Question = ({question}) => {
```

Comment:
fair points. i will add a comment. however, this is the way i am creating components these days as it removes any questions of state, props, etc and by definition makes this a slave component to the parent container.